dictionaries/actors.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Adagio": "*Adagio*",
+  "Alpha": "*Alpha*",
+  "Ardan": "*Ardan*",
```

Comment:
quick question if we change these to a dictionary shouldn't the `key` represent the value given by the api? So `*Ardan*` should be the Key and the friendlyName that the dictionary returns should be the `Ardan`

Reply:
Wow, yeah totally should.